Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData/MemoryDatabase/AirportMemoryDatabase.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Ibistic.Public.OpenAirportData.MemoryDatabase
+{
+    public class AirportIataCodeDatabase
+    {
+        private readonly ConcurrentDictionary<string, Airport> _airports = new ConcurrentDictionary<string, Airport>();
```

Comment:
It would perhaps make sense to make the IATA key case insensitive, depending on whether you want to support case invariant lookups. Also, it might make sense to call it _airtportsByIataCode or something.

Reply:
Agreed. Fixing.